source/MaterialXFormat/Util.cpp

```diff
@@ -222,4 +222,21 @@ FileSearchPath getSourceSearchPath(ConstDocumentPtr doc)
     return searchPath;
 }
 
+FileSearchPath getDefaultLibraryPath(const FilePath& startPath)
```

Comment:
This looks like a great direction!  To make the parallels between C++ and Python more clear, perhaps we should call this method `getDefaultDataSearchPath` and omit the input argument?  What are your thoughts?

Reply:
I think we can do this:  Call this `getDefaultDataSearchPath()`  with a default argument of an empty path.
If empty then the  module path would be used.

Unlike Python where we always know where to look, I think we'd still want to allow users to specify where to start from instead of assuming the libraries reside within the same folder hierarchy as the binary as the user can create their binaries anywhere.